docs/install.rst

```diff
@@ -151,12 +151,15 @@ documentation <https://pip.pypa.io/en/stable/user_guide/#user-installs>`_.
 {% if is_development %}
 
 Alternatively, if you intend to do development on other software that uses
-``astropy``, such as an affiliated package, consider installing ``astropy`` into a :ref:`virtualenv<using-virtualenv>`.
+``astropy``, such as an affiliated package, consider installing ``astropy``
+into a :ref:`virtualenv<virtual_envs>`.
```

Comment:
If this links to RST inside `development`, you might need to hardcode the URL due to #11342 . To be sure, you can tweak `docs/conf.py` to mimic a stable build locally and make sure that has no warnings. See #11344 for example.

Reply:
Here I was planning to rebase this on top of my fix for #11366.  For which I see I haven't made a PR yet; I thought I already did.

Reply:
Should I withhold approval until the rebase is done?

Reply:
Yes please, that would make sense.  (It is done now, see #11622 )